[2024-07-02T18:42:24.369+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T18:42:24.389+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T18:42:24.393+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T18:42:24.445+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T18:42:24.463+0000] {standard_task_runner.py:60} INFO - Started process 97 to run task
[2024-07-02T18:42:24.471+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp42tui_9q']
[2024-07-02T18:42:24.494+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask language
[2024-07-02T18:42:24.630+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 5b7251376729
[2024-07-02T18:42:25.745+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T18:42:25.754+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: process_table() missing 1 required positional argument: 'table_type'
[2024-07-02T18:42:25.768+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T184224, end_date=20240702T184225
[2024-07-02T18:42:25.836+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 10 for task language (process_table() missing 1 required positional argument: 'table_type'; 97)
[2024-07-02T18:42:25.877+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T18:42:25.983+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T18:50:28.298+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T18:50:28.310+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T18:50:28.312+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T18:50:28.346+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T18:50:28.355+0000] {standard_task_runner.py:60} INFO - Started process 87 to run task
[2024-07-02T18:50:28.360+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpuuzfs8hb']
[2024-07-02T18:50:28.369+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask language
[2024-07-02T18:50:28.491+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host e236fd1115af
[2024-07-02T18:50:29.143+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T18:50:29.148+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T18:50:29.149+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T18:50:29.154+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T18:50:29.999+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-07-02T18:50:30.009+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T185028, end_date=20240702T185030
[2024-07-02T18:50:30.057+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-07-02T18:50:30.081+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:20:13.116+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:20:13.157+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:20:13.162+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:20:13.199+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:20:13.216+0000] {standard_task_runner.py:60} INFO - Started process 98 to run task
[2024-07-02T20:20:13.236+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp1tvi4hn8']
[2024-07-02T20:20:13.245+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask language
[2024-07-02T20:20:13.435+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 881b4526a2de
[2024-07-02T20:20:14.544+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:20:14.550+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:20:14.557+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:20:14.561+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:20:14.627+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag.py", line 22, in process_table
    return  execute(table, stage, "FULL","public")
  File "/app/main.py", line 25, in execute
    SqlToStaging().create_full().some_function(stage, type, schema, table)
  File "/app/concretestage/SQLToStaging.py", line 29, in some_function
    print(get_data_from_conf_table("staff", "SqlToStaging"))
  File "/app/myFramework/utils/utils.py", line 137, in get_data_from_conf_table
    cur = conn.getCursor("Postgres")
  File "/app/myFramework/source/posgresql/connect.py", line 6, in getCursor
    conn = psycopg2.connect(database = dbname,
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "postgres_db" (172.20.0.3), port 9999 failed: FATAL:  database "Postgres" does not exist

[2024-07-02T20:20:14.761+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T202013, end_date=20240702T202014
[2024-07-02T20:20:14.851+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 10 for task language (connection to server at "postgres_db" (172.20.0.3), port 9999 failed: FATAL:  database "Postgres" does not exist
; 98)
[2024-07-02T20:20:14.892+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T20:20:15.011+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:22:14.208+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:22:14.222+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:22:14.231+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:22:14.257+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:22:14.288+0000] {standard_task_runner.py:60} INFO - Started process 97 to run task
[2024-07-02T20:22:14.304+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmphx2z8ngr']
[2024-07-02T20:22:14.341+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask language
[2024-07-02T20:22:14.484+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host b2ca66e4d1f6
[2024-07-02T20:22:15.671+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:22:15.702+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:22:15.730+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:22:15.744+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:22:16.082+0000] {logging_mixin.py:188} INFO -    0      1          2       3     4   ...    10    11    12            13    14
0   9  staff  dvdrental  public  full  ...  None  None  None  SqlToStaging  None

[1 rows x 15 columns]
[2024-07-02T20:22:16.088+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-07-02T20:22:16.164+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T202214, end_date=20240702T202216
[2024-07-02T20:22:16.204+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-07-02T20:22:16.290+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:25:37.697+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:25:37.716+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:25:37.724+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:25:37.754+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:25:37.777+0000] {standard_task_runner.py:60} INFO - Started process 101 to run task
[2024-07-02T20:25:37.784+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp_uecn409']
[2024-07-02T20:25:37.807+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask language
[2024-07-02T20:25:38.015+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host e847a37dafab
[2024-07-02T20:25:39.228+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:25:39.231+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:25:39.234+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:25:39.237+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:25:39.298+0000] {logging_mixin.py:188} INFO - <class 'pandas.core.frame.DataFrame'>
[2024-07-02T20:25:39.390+0000] {logging_mixin.py:188} INFO -    id sourcetablename sourcedbname  ...  code         stage last_run_date
0   9           staff    dvdrental  ...  None  SqlToStaging          None

[1 rows x 15 columns]
[2024-07-02T20:25:39.392+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-07-02T20:25:39.422+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T202537, end_date=20240702T202539
[2024-07-02T20:25:39.473+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-07-02T20:25:39.557+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:33:53.551+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:33:53.582+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:33:53.595+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:33:53.638+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:33:53.644+0000] {standard_task_runner.py:60} INFO - Started process 87 to run task
[2024-07-02T20:33:53.652+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpv0zcb37h']
[2024-07-02T20:33:53.671+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask language
[2024-07-02T20:33:53.823+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 99fbc1087931
[2024-07-02T20:33:55.297+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:33:55.306+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:33:55.314+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:33:55.314+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:33:55.347+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3653, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 147, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 176, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7080, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7088, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'SourceDBName'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag.py", line 22, in process_table
    return  execute(table, stage, "FULL","public")
  File "/app/main.py", line 25, in execute
    SqlToStaging().create_full().some_function(stage, type, schema, table)
  File "/app/concretestage/SQLToStaging.py", line 30, in some_function
    print(f"{df['SourceDBName']} \n {df['SourceTableName']} \n {df['SourceSchema']}")
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/frame.py", line 3761, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3655, in get_loc
    raise KeyError(key) from err
KeyError: 'SourceDBName'
[2024-07-02T20:33:55.366+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T203353, end_date=20240702T203355
[2024-07-02T20:33:55.382+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 3 for task language ('SourceDBName'; 87)
[2024-07-02T20:33:55.436+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T20:33:55.462+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:36:07.797+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:36:07.815+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:36:07.821+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:36:07.852+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:36:07.864+0000] {standard_task_runner.py:60} INFO - Started process 92 to run task
[2024-07-02T20:36:07.872+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp_q9fknmv']
[2024-07-02T20:36:07.886+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask language
[2024-07-02T20:36:08.044+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 8ef3b2afa95a
[2024-07-02T20:36:08.698+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:36:08.701+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:36:08.701+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:36:08.704+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:36:08.748+0000] {logging_mixin.py:188} INFO - 0    dvdrental
Name: sourcedbname, dtype: object 
 0    staff
Name: sourcetablename, dtype: object 
 0    public
Name: sourceschema, dtype: object
[2024-07-02T20:36:08.750+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-07-02T20:36:08.779+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T203607, end_date=20240702T203608
[2024-07-02T20:36:08.867+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-07-02T20:36:08.892+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:40:31.096+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:40:31.120+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:40:31.123+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:40:31.146+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:40:31.153+0000] {standard_task_runner.py:60} INFO - Started process 90 to run task
[2024-07-02T20:40:31.167+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp2dn2ecm6']
[2024-07-02T20:40:31.176+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask language
[2024-07-02T20:40:31.499+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host f7a45131ad00
[2024-07-02T20:40:32.367+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:40:32.370+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:40:32.378+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:40:32.385+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:40:32.475+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag.py", line 22, in process_table
    return  execute(table, stage, "FULL","public")
  File "/app/main.py", line 25, in execute
    SqlToStaging().create_full().some_function(stage, type, schema, table)
  File "/app/concretestage/SQLToStaging.py", line 29, in some_function
    df = get_data_from_conf_table(table, stage)
  File "/app/myFramework/utils/utils.py", line 142, in get_data_from_conf_table
    df.columns = [desc[0] for desc in cur.description]
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 6002, in __setattr__
    return object.__setattr__(self, name, value)
  File "pandas/_libs/properties.pyx", line 69, in pandas._libs.properties.AxisProperty.__set__
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 730, in _set_axis
    self._mgr.set_axis(axis, labels)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/managers.py", line 225, in set_axis
    self._validate_set_axis(axis, new_labels)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/base.py", line 70, in _validate_set_axis
    raise ValueError(
ValueError: Length mismatch: Expected axis has 0 elements, new values have 15 elements
[2024-07-02T20:40:32.581+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T204031, end_date=20240702T204032
[2024-07-02T20:40:32.633+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 4 for task language (Length mismatch: Expected axis has 0 elements, new values have 15 elements; 90)
[2024-07-02T20:40:32.696+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T20:40:32.718+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:44:45.176+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:44:45.208+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:44:45.211+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:44:45.252+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:44:45.281+0000] {standard_task_runner.py:60} INFO - Started process 100 to run task
[2024-07-02T20:44:45.287+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpuvlwzpyn']
[2024-07-02T20:44:45.293+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask language
[2024-07-02T20:44:45.458+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 28c1247628aa
[2024-07-02T20:44:46.738+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:44:46.741+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:44:46.743+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:44:46.748+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 type: FULL 
 schema: public 
 table:language
[2024-07-02T20:44:46.750+0000] {logging_mixin.py:188} INFO - sqltostaging language
[2024-07-02T20:44:46.753+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-07-02T20:44:46.767+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T204445, end_date=20240702T204446
[2024-07-02T20:44:46.823+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-07-02T20:44:46.840+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:47:20.719+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:47:20.746+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:47:20.752+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:47:20.779+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:47:20.792+0000] {standard_task_runner.py:60} INFO - Started process 92 to run task
[2024-07-02T20:47:20.798+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmphbl6rc3d']
[2024-07-02T20:47:20.807+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask language
[2024-07-02T20:47:20.973+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 0dbf5a897656
[2024-07-02T20:47:22.145+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:47:22.160+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:47:22.169+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:47:22.173+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag.py", line 22, in process_table
    return  execute(table, stage)
TypeError: execute() missing 2 required positional arguments: 'type' and 'schema'
[2024-07-02T20:47:22.189+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T204720, end_date=20240702T204722
[2024-07-02T20:47:22.215+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 9 for task language (execute() missing 2 required positional arguments: 'type' and 'schema'; 92)
[2024-07-02T20:47:22.291+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T20:47:22.457+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:49:58.189+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:49:58.227+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:49:58.233+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:49:58.273+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:49:58.280+0000] {standard_task_runner.py:60} INFO - Started process 90 to run task
[2024-07-02T20:49:58.304+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpsri3lbmc']
[2024-07-02T20:49:58.315+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask language
[2024-07-02T20:49:58.503+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 82e116e15bed
[2024-07-02T20:49:59.827+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:49:59.835+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:49:59.836+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:49:59.837+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 table:language
[2024-07-02T20:49:59.837+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag.py", line 22, in process_table
    return  execute(table, stage)
  File "/app/main.py", line 25, in execute
    SqlToStaging().create_full().some_function(stage, table)
TypeError: some_function() missing 2 required positional arguments: 'schema' and 'table'
[2024-07-02T20:49:59.852+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T204958, end_date=20240702T204959
[2024-07-02T20:49:59.873+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 10 for task language (some_function() missing 2 required positional arguments: 'schema' and 'table'; 90)
[2024-07-02T20:49:59.927+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T20:49:59.992+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:51:51.356+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:51:51.430+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:51:51.434+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:51:51.494+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:51:51.508+0000] {standard_task_runner.py:60} INFO - Started process 101 to run task
[2024-07-02T20:51:51.518+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpjxudvohf']
[2024-07-02T20:51:51.531+0000] {standard_task_runner.py:88} INFO - Job 17: Subtask language
[2024-07-02T20:51:51.636+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host 910796d1882f
[2024-07-02T20:51:52.867+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:51:52.882+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:51:52.888+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:51:52.891+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 table:language
[2024-07-02T20:51:52.953+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag.py", line 22, in process_table
    return  execute(table, stage)
  File "/app/main.py", line 25, in execute
    SqlToStaging().create_full().some_function(stage, table)
  File "/app/concretestage/SQLToStaging.py", line 30, in some_function
    df = get_data_from_conf_table(table, stage)
  File "/app/myFramework/utils/utils.py", line 142, in get_data_from_conf_table
    df.columns = [desc[0] for desc in cur.description]
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 6002, in __setattr__
    return object.__setattr__(self, name, value)
  File "pandas/_libs/properties.pyx", line 69, in pandas._libs.properties.AxisProperty.__set__
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 730, in _set_axis
    self._mgr.set_axis(axis, labels)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/managers.py", line 225, in set_axis
    self._validate_set_axis(axis, new_labels)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/base.py", line 70, in _validate_set_axis
    raise ValueError(
ValueError: Length mismatch: Expected axis has 0 elements, new values have 15 elements
[2024-07-02T20:51:52.986+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T205151, end_date=20240702T205152
[2024-07-02T20:51:53.008+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 17 for task language (Length mismatch: Expected axis has 0 elements, new values have 15 elements; 101)
[2024-07-02T20:51:53.056+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-02T20:51:53.083+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T20:54:17.447+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:54:17.477+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T20:54:17.477+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-07-02T20:54:17.511+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): language> on 2024-07-01 00:00:00+00:00
[2024-07-02T20:54:17.541+0000] {standard_task_runner.py:60} INFO - Started process 89 to run task
[2024-07-02T20:54:17.554+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sqltostaging', 'language', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp64hk3gxn']
[2024-07-02T20:54:17.572+0000] {standard_task_runner.py:88} INFO - Job 6: Subtask language
[2024-07-02T20:54:17.838+0000] {task_command.py:423} INFO - Running <TaskInstance: sqltostaging.language scheduled__2024-07-01T00:00:00+00:00 [running]> on host df9727a9ba29
[2024-07-02T20:54:18.030+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sqltostaging' AIRFLOW_CTX_TASK_ID='language' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T20:54:18.038+0000] {logging_mixin.py:188} INFO - dvtobv :dag_id
[2024-07-02T20:54:18.044+0000] {logging_mixin.py:188} INFO - table: language
[2024-07-02T20:54:18.051+0000] {logging_mixin.py:188} INFO - 
 stage: sqltostaging 
 table:language
[2024-07-02T20:54:18.916+0000] {logging_mixin.py:188} INFO - 0    dvdrental
Name: sourcedbname, dtype: object 
 0    staff
Name: sourcetablename, dtype: object 
 0    public
Name: sourceschema, dtype: object
[2024-07-02T20:54:18.917+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-07-02T20:54:18.949+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sqltostaging, task_id=language, execution_date=20240701T000000, start_date=20240702T205417, end_date=20240702T205418
[2024-07-02T20:54:19.083+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-07-02T20:54:19.125+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
